src/Agent.Worker/Release/Artifacts/BuildArtifact.cs

```diff
@@ -99,6 +99,10 @@ public IArtifactDetails GetArtifactDetails(IExecutionContext context, AgentArtif
             var tfsUrl = context.Variables.Get(WellKnownDistributedTaskVariables.TFCollectionUrl);
 
             Guid projectId = context.Variables.System_TeamProjectId ?? Guid.Empty;
+            if(artifactDetails.ContainsKey("Project")){
```

Comment:
use style we use in c# for if block (this one resembles TS files)

if (XXX)
{
...
}